src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import styled, { DefaultTheme } from "styled-components";
+import CheckmarkCircleIcon from "../Svg/Icons/CheckmarkCircle";
+import ErrorIcon from "../Svg/Icons/Error";
+import BlockIcon from "../Svg/Icons/Block";
+import InfoIcon from "../Svg/Icons/Info";
+import { Text } from "../Text";
+import { AlertProps, variants } from "./types";
+
+interface ThemedIconLabel {
+  variant: AlertProps["variant"];
+  theme: DefaultTheme;
+}
+
+const getThemeColor = ({ theme, variant = variants.INFO }: ThemedIconLabel) => {
+  switch (variant) {
+    case variants.DANGER:
+      return theme.colors.failure;
+    case variants.WARNING:
+      return theme.colors.warning;
+    case variants.SUCCESS:
+      return theme.colors.success;
+    case variants.INFO:
+    default:
+      return theme.colors.secondary;
+  }
+};
+
+const getIcon = (variant: AlertProps["variant"] = variants.INFO) => {
+  switch (variant) {
+    case variants.DANGER:
+      return BlockIcon;
+    case variants.WARNING:
+      return ErrorIcon;
+    case variants.SUCCESS:
+      return CheckmarkCircleIcon;
+    case variants.INFO:
+    default:
+      return InfoIcon;
+  }
+};
+
+const Title = styled(Text)`
+  font-weight: 600;
```

Comment:
you can just active the prop `bold` using .attrs, to keep consistency in case we change the bold style on Text